app/components/Task.js

```diff
@@ -5,12 +5,31 @@ import ContentAdd from 'material-ui/svg-icons/content/add';
 import ContentDeleteSweep from 'material-ui/svg-icons/content/delete-sweep';
 import TextField from 'material-ui/TextField/TextField';
 import Checkbox from 'material-ui/Checkbox/Checkbox';
-import { ListItem } from 'material-ui/List';
 import PropTypes from 'prop-types';
 
+import TimeMeasurable from './task_decorators/TimeMeasurable';
+import Octcatable from './task_decorators/Octcatable';
+
 import styles from './Task.css';
 
 class Task extends Component {
+  static propTypes = {
```

Comment:
if we are using propTypes then we might not need flow.

Reply:
also flow requires `// @flow` at start of every files

Reply:
oh!